src/Analysis/AnalysisApp.jsx

```diff
@@ -73,6 +75,10 @@ class AnalysisApp extends React.Component {
       return (
         <ReduxGWASApp />
       );
+    case 'GWASUIMockup':
```

Comment:
If this app is no longer a mockup, let's change its name accordingly in all the places that is appropriate

Reply:
renamed to GWASUIApp